components/edit-collective/sections/Tiers.js

```diff
@@ -1,597 +1,189 @@
 import React from 'react';
```

Comment:
To make it clear (the diff is not):
- `components/edit-collective/sections/Tiers.js` -> `components/edit-collective/sections/TiersLegacy.js`
- `components/edit-collective/sections/TiersRevamp.js` -> `components/edit-collective/sections/Tiers.js`